VAMobile/src/store/api/types/auth.ts

```diff
@@ -8,13 +8,11 @@ export enum AUTH_STORAGE_TYPE {
   NONE = 'NONE',
 }
 
-export type LoginServiceTypes = 'IAM' | 'SIS'
+export type LoginServiceTypes = 'SIS'
```

Comment:
at the moment we do not, but I didn't want to drop that as I wasn't sure if any login changes were going to happen. This might be a good thing to address when we redo login, but should be find to keep for this ticket.